tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -99,12 +99,7 @@ ROOT::Experimental::RNTupleReader::RNTupleReader(std::unique_ptr<ROOT::Experimen
                                                  std::unique_ptr<ROOT::Experimental::Detail::RPageSource> source)
    : fSource(std::move(source)), fModel(std::move(model)), fMetrics("RNTupleReader")
 {
-   if (!fSource) {
-      throw RException(R__FAIL("null source"));
-   }
-   if (!fModel) {
-      throw RException(R__FAIL("null model"));
-   }
```

Comment:
I think we have to retain the model check, or move it where the user can provide one

Reply:
as discussed, we remove the checks here